hippunfold/workflow/rules/autotop.smk

```diff
@@ -69,14 +101,20 @@ rule create_unfold_coord_map:
     shell:
         'c3d {input.nii} -cmp -omc {output.nii}'
 
-        
+def get_io(wildcards):
```

Comment:
is this used anywhere?

Reply:
in any case this would be the place to make use of the config parameter and pick the laminar_coords_method, and would presumably then be called in the create_warps input

Reply:
fixed